exchanges/coinbene/coinbene_test.go

```diff
@@ -0,0 +1,191 @@
+package coinbene
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/thrasher-corp/gocryptotrader/config"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+)
+
+// Please supply your own keys here for due diligence testing
+const (
+	testAPIKey              = ""
+	testAPISecret           = ""
+	canManipulateRealOrders = false
+
+	btcusdt = "BTC/USDT"
+)
+
+var c Coinbene
+var setupRan bool
+var m sync.Mutex
+
+func TestSetup(t *testing.T) {
```

Comment:
We've started switching over to TestMain for exchange package testing so there's no need for a sync.Mutex and calling TestSetup(). Can also use t.Parrallel() on these tests.